src/Employer/Employer.Web/Views/Dashboard/Dashboard.cshtml

```diff
@@ -105,7 +105,7 @@
                         </a>
                         <span asp-hide="@Model.HasLiveVacancy">@FilteringOptions.Live @Model.VacancyTextLive</span>
                     </h3>
-                    <span class="govuk-body das-card--description">Vacancies on Find an apprenticeship service.</span>
+                    <span class="govuk-body das-card--description">Your adverts on the Find anapprenticeship service.</span>
```

Comment:
Should there be a space between anapprenticeship?

Reply:
yes, I've updated